### ISSUE TITLE
Reduce the size of the image

### ISSUE THREAD

**Issue by Yes21** [RespawnDespair/wifibroadcast-image-builder#59]
Images produced by the building script have a size of 3.7 Gb, but regular 1.6RC6 only 1.7 Gb.
It takes many time to burn a 3.7 Gb on 2 µSD card (even class 10) each time one wants to try a new version !!
The rootfs partition could be reduced from 3.6 Gb to 2.4 Gb (perhaps less ?), and the user could expand it after the first boot.

**Comment by rodizio1** [comment]
Yes, could be reduced. 

1.7GB like the orginal 1.6RC6 may not be possible because we now use a newer Raspbian distribution version and usually, distributions get larger with every version.

With buildroot it would be a lot smaller, I'd say well below 500MB.

**Comment by Yes21** [comment]
@careyer has just added a new [wiki page](https://github.com/bortek/EZ-WifiBroadcast/wiki/Expert-%7E-Shrinking-Images) .
Using this script, I could reduce the image size from 3.7 Gb to 3.1 Gb. I thought it could be smaller but ...

**Comment by rodizio1** [comment]
590MB is not possible without deleting or compressing stuff, just the files inside the image (without lost space due to file system overhead) are already 1.something gigs.

**Comment by Yes21** [comment]
@careyer I tried without parameter.

**Comment by careyer** [comment]
@rodizio1 : Yes sorry.. I was wrong. Without compression it was indeed 1.7GB.... 590MB was after compression. (just downloaded it from my cloud storage).  @Yes21: thanks for trying and confirming. Saving 600MB does not make a whole lot of a difference. Removing the wiki site.
